File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/DataPacket.java
package org.firstinspires.ftc.teamcode.util;

import com.acmerobotics.dashboard.telemetry.TelemetryPacket;

public class DataPacket extends TelemetryPacket {

    public void addData(String key, Object val) {
        addLine(key + ": " + val.toString());
    }

    public void addSpace() {
        addLine(" ");
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/DataPacket.kt
package org.firstinspires.ftc.teamcode.util

import com.acmerobotics.dashboard.telemetry.TelemetryPacket

class DataPacket : TelemetryPacket() {
    fun addData(key: String, `val`: Any) = addLine("$key: $`val`")
    fun addSpace() = addLine(" ")
}